index.html

```diff
@@ -74,7 +74,10 @@ <H2 id="external">Users</H2>
 <ol>
 
 <!-- 2018 -->
-
+	
+<li>Wang Jie，Tong Jingyu，Gao Qinghua，Wu Zhenyu, Bi Sheng, Wang Hongyu.<br>
```

Comment:
Thank you!!

please alphabetize this, and follow the existing structure. link via DOI rather than directly to publisher, newlines, etc.